### PR TITLE
Python3 port for openclpow.py, test_openclpow.py

### PR DESCRIPTION
**Pull request by kdcis**
### Enabled Python3

### CODE REVIEW

File: src/openclpow.py [hunk 1]
```diff
@@ -5,9 +5,14 @@
 import os
 from struct import pack
 
-import paths
-from bmconfigparser import BMConfigParser
-from state import shutdown
+try:
```
Comment (PeterSurda) [inline src/openclpow.py:8]: Again I need to investigate this like in other PRs.

**Review by PeterSurda — COMMENTED** [review]
I need to think about imports, rest is ok.

**Comment by g1itch** [comment]
Do you have an AMD graphics card? Did you ran the test?

**Comment by PeterSurda** [comment]
@g1itch The servers running the buildbot workers all have AMD GPUs or APUs, but the VMs/containers don't have access to it, yet. For the time being, I could run the test manually just to check.

**Comment by g1itch** [comment]
> @g1itch The servers running the buildbot workers all have AMD GPUs or APUs, but the VMs/containers don't have access to it, yet. For the time being, I could run the test manually just to check.

cool, I've already started to think about buying some PCI-X GPU...

**Review by PeterSurda — APPROVED** [review]

**Comment by g1itch** [comment]
Bad signatures! This wont merge.

**Review by g1itch — CHANGES_REQUESTED** [review]
Bad signature!

**Comment by kdcis** [comment]
> Bad signature!

Signature updated.

**Review by g1itch — CHANGES_REQUESTED** [review]
Please, rebase

**Review by g1itch — APPROVED** [review]

**Pull request merged**